solana/src/main/java/com/solana/vendor/bip39/WordList.kt

```diff
@@ -0,0 +1,2082 @@
+package com.solana.vendor.bip39
+
+import java.lang.UnsupportedOperationException
+import java.util.*
+
+class WordList internal constructor(val languageCode: String) {
+    constructor(locale: Locale = Locale.ENGLISH) : this(locale.language)
+
+    init {
+        validate(languageCode)
```

Comment:
the only thing I don't like here is that `WordList` might throw and exception on initialization, which might not be expected by the caller. I think we should tag the constructor here with @Throws. (And maybe also the `validate` method?)